docs/topics/Permissions.md

```diff
@@ -33,15 +33,15 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | BAN_MEMBERS \*                | `0x0000000000000004` `(1 << 2)`  | Allows banning members                                                                                                                              |              |
 | ADMINISTRATOR \*              | `0x0000000000000008` `(1 << 3)`  | Allows all permissions and bypasses channel permission overwrites                                                                                   |              |
 | MANAGE_CHANNELS \*            | `0x0000000000000010` `(1 << 4)`  | Allows management and editing of channels                                                                                                           | T, V, S      |
-| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management and editing of the guild                                                                                                          |              |
+| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management of the server name, region, and adding bots to the server                                                                          |              |
```

Comment:
server region doesn't exist anymore, and manage server does a lot more than what is listed here.

Reply:
```suggestion
| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows editing guild's settings and welcome screen, managing guild template, integrations and widget and deleting invites                           |              |
```

Reply:
Fair. I'm updating that in my latest commit involving most of the suggestions made here

Reply:
I would rather err on the side of being more specific then more vague, let me know if there is anything still missing from manage server in the latest commit.